qa/tasks/cephfs/fuse_mount.py

```diff
@@ -305,6 +305,8 @@ def umount_wait(self, force=False, require_clean=False, timeout=900):
         """
         :param force: Complete cleanly even if the MDS is offline
         """
+        assert self.is_mounted() and self.fuse_daemon
```

Comment:
Won't this just raise an assertion that kills the DaemonWatchdog too? We probably just want to return early if it's not mounted or raise a specific NotMounted exception the DaemonWatchdog can catch.

Reply:
Let me check more on that. I see different options to deal with this situation. I see the [kernel mount](https://github.com/ceph/ceph/blob/f90e3e102659c7930222a213e86f19d5caf736cd/qa/tasks/cephfs/kernel_mount.py#L137) does a return.

Reply:
@batrick Fixed. Please check.